contracts/eosiolib/checktime.h

```diff
@@ -0,0 +1,19 @@
+/**
+ *  @file
+ *  @copyright defined in eos/LICENSE.txt
+ */
+#pragma once
+
+#include <eosiolib/system.h>
+
+extern "C" {
+  /**
+   *  @defgroup checktime API
+   *  @brief Declares checktime function. 
+   *  @ingroup checktimeapi
+   *
+   *  @{
+   */
+   void checktime();
```

Comment:
I think its preferable to not treat `chectime` as part of the ABI.  I don't think we want contracts calling this directly. @spoonincode may want to weigh in.

Reply:
Yeah I don't think we should expose it (we have a ticket somewhere where we intend to actively refuse linkage to it in the future). This call is implementation detail.

Reply:
Yeah, this was an oversight on my part.  I exposed it so that I could see what it was actually doing and forgot to delete it from the header.